### PR TITLE
Added 2 manual tests (calibration and tactile sensors)

### LINKED ISSUE #13: Visual check of calibration

### LINKED ISSUE #15: Tactile Sensors
this one may also require human intervention, unless we can have the fingertips touch each other. It could be a combo position/tactile test?

### PR DESCRIPTION
**Pull request by ugocupcic**
Those tests are run by a human checking visually whether things work or pressing on the tactiles. The GUI plugin has also been edited.

fixes #13 
fixes #15.

### CODE REVIEW

File: sr_self_test/include/sr_self_test/manual_test.hpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @brief
```
Comment (markpitchless) [inline sr_self_test/include/sr_self_test/manual_test.hpp:21]: Would be nice to have something to intro the class.

File: sr_self_test/src/sr_self_test.cpp [hunk 2]
```diff
@@ -262,6 +269,15 @@
     safe_target.joint_target = 0.0;
     safe_targets_->insert( std::pair<std::string, sr_robot_msgs::joint>(safe_target.joint_name, safe_target) );
   }
+
+  //////
+  // Add Manual Tests
+
+  void SrSelfTest::add_manual_tests_()
```
Comment (markpitchless) [inline sr_self_test/src/sr_self_test.cpp:276]: Not sure this method is really needed, just seems to be wrapping addManualTests with the same name but formatted differently.
Callers could just do test_runner_.addManualTests(); ?

File: sr_self_test/test/automatic_manual_test_response.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env python
+
+import roslib; roslib.load_manifest("sr_self_test")
+import rospy
+from sr_robot_msgs.srv import ManualSelfTest, ManualSelfTestResponse
+
+class AutomaticResponse(object):
```
Comment (markpitchless) [inline sr_self_test/test/automatic_manual_test_response.py:7]: Nice :)

**Comment by ugocupcic** [comment]
Fixed the issues in branch B_self_test_small_fixes

**Pull request merged**